app/src/main/java/lt/vilnius/tvarkau/entity/Profile.java

```diff
@@ -59,6 +62,10 @@ public void setMobilePhone(String mobilePhone) {
         this.mobilePhone = mobilePhone;
     }
 
+    public String getPersonalCode() {
+        return personalCode;
+    }
+
     public String createJsonData() {
```

Comment:
See `equals` method. Or better convert to Kotlin data class to avoid such mistakes